### ISSUE TITLE
[TF2] "Duck Journal" UI blocking styles

### ISSUE THREAD

**Issue by MN-Discord** [ValveSoftware/Source-1-Games#6352]
**Description**
The different styles of the "Duck Journal" are blocked by the Select style view model UI making it super hard to switch between the different styles

**Reproduction steps**
Simply try changing the style on the duck journal [atm unknown if this bug is caused by any in-game settings]

**Additional files**
https://youtu.be/FylNvlG3KQs

**Comment by JoriKos** [comment]
![afbeelding](https://github.com/user-attachments/assets/e2af12a6-414f-42a1-b751-e3ea1058a3e9)

It's fine on my end, are you sure you have no custom stuff interfering? Are you using the scaling UI branch?

**Comment by MN-Discord** [comment]
> ![afbeelding](https://private-user-images.githubusercontent.com/43441605/368148429-e2af12a6-414f-42a1-b751-e3ea1058a3e9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY1ODkzODIsIm5iZiI6MTcyNjU4OTA4MiwicGF0aCI6Ii80MzQ0MTYwNS8zNjgxNDg0MjktZTJhZjEyYTYtNDE0Zi00MmExLWI3NTEtZTNlYTEwNThhM2U5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTE3VDE2MDQ0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYyYzJmYmYyODkwMzZkNmUwZmY5N2Q1NWQxMzY4YThkMmMzYWM1MTIwYzRlYmY2ZWM3YmRjNmU1YzczMjAyNjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.kqZFWZtH572176S47bi-kD6IV4peKmJABRqZgT8C4QQ)
> 
> It's fine on my end, are you sure you have no custom stuff interfering? Are you using the scaling UI branch?

I dont believe so? the only files i have manually added into my game are custom maps, I literally have 0 things in my custom except the base stuff. 
Also I just booted up my game and it fixed itself? I think you have to do something in-game for it to happen ig?

**Comment by KimmyTF2** [comment]
I've tried everything to reproduce this. Changing resolutions, changing quality, alt-tabbing, changing between maps with different UI elements visible at once, but can't reproduce it.
